projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/checkout-group.actions.migration.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  CHECKOUT_ACTIONS,
+  PLACE_ORDER_ACTION,
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CHECKOUT_GROUP_ACTIONS_MIGRATIONS: MethodPropertyDeprecation[] = [
+  {
+    class: CHECKOUT_ACTIONS,
+    importPath: SPARTACUS_CORE,
+    deprecatedNode: PLACE_ORDER_ACTION,
```

Comment:
As Patrick stated in my PR, let's change the variable name to `PLACE_ORDER_CLASS` to stay consistent with the previous old one